docs/collaboration.md

```diff
@@ -0,0 +1,105 @@
+# Collaborative Publishing with Posit Publisher
+
+[Posit Publisher](https://github.com/posit-dev/publisher)—the extension to
+publish content to Posit Connect using
+[Visual Studio Code](https://code.visualstudio.com/) or
+[Positron](https://positron.posit.co/)—enables seemless collaboration. This
+workflow guide explains how to collaborate with others on a deployed piece of
+content.
+
+## Deployment and Configuration files
+
+Posit Publisher creates and utilizes two files for deploying content. Both are
+inside of the `.posit/publish` directory in the same directory as
+your entrypoint file.
```

Comment:
Should we mention here about adding these to version control?

Reply:
I mentioned it at the beginning of the source control section, but we could definitely move it up. I tried to keep that separate to avoid bouncing any user that wasn't using source control.